spec/factories/tasks.rb

```diff
@@ -5,5 +5,10 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    trait :complete do
+      status { 'done' }
```

Comment:
`String`ではなく`Symbol`で記載した方が`enum`の値であることが伝わりますね！
```suggestion
      status { :done }
```

Reply:
5db8eb2354b1cbc71581bc4d80bc53e2fd5b7d07 表記方法を`symbol`へ修正しました。